bluez.yaml

```diff
@@ -93,11 +93,12 @@ subpackages:
           mkdir -p ${{targets.subpkgdir}}/usr/bin
           mv ${{targets.destdir}}/usr/bin/meshctl ${{targets.subpkgdir}}/usr/bin
 
-  - name: bluez-plugins
-    pipeline:
-      - runs: |
-          mkdir -p ${{targets.subpkgdir}}/usr/lib/bluetooth/plugins
-          mv ${{targets.destdir}}/usr/lib/bluetooth/plugins/sixaxis.so ${{targets.subpkgdir}}/usr/lib/bluetooth/plugins
+  # Removed due to https://git.alpinelinux.org/aports/commit/?id=cea6812f3eeb12f311b6eb5590b2b3379d4743f1
+  #- name: bluez-plugins
+  #  pipeline:
+  #    - runs: |
+  #        mkdir -p ${{targets.subpkgdir}}/usr/lib/bluetooth/plugins
+  #        mv ${{targets.destdir}}/usr/lib/bluetooth/plugins/sixaxis.so ${{targets.subpkgdir}}/usr/lib/bluetooth/plugins
```

Comment:
Should we just delete these lines and keep the note about them in the PR, for any future archeologists?

Reply:
Understood, I have already removed the commented lines.